test/modules/events.js

```diff
@@ -499,9 +499,9 @@
             assessGestureExpectations({
                 description:           'enter-nonprimarypress-nonprimaryrelease-exit (secondary/right click):  ',
                 enterCount:            1,
-                exitCount:             1,
+                exitCount:             2,
```

Comment:
So with the addition of this patch, we can lose the hover gesture point until the next mouse move? Does seem like it might be worth addressing that.